be/src/exec/exec_node.cpp

```diff
@@ -672,13 +675,51 @@ void ExecNode::eval_conjuncts(const std::vector<ExprContext*>& ctxs, vectorized:
 void ExecNode::eval_join_runtime_filters(vectorized::Chunk* chunk) {
     if (chunk == nullptr) return;
     _runtime_filter_collector.evaluate(chunk);
+    eval_filter_null_values(chunk);
 }
 
 void ExecNode::eval_join_runtime_filters(vectorized::ChunkPtr* chunk) {
     if (chunk == nullptr) return;
     eval_join_runtime_filters(chunk->get());
 }
 
+void ExecNode::eval_filter_null_values(vectorized::Chunk* chunk) {
+    if (_filter_null_value_columns.size() == 0) return;
+    size_t before_size = chunk->num_rows();
+    if (before_size == 0) return;
+
+    // lazy allocation.
+    vectorized::Buffer<uint8_t> selection(0);
+
+    for (SlotId slot_id : _filter_null_value_columns) {
+        const ColumnPtr& c = chunk->get_column_by_slot_id(slot_id);
+        const vectorized::NullableColumn* nullable_column =
+                vectorized::ColumnHelper::as_raw_column<vectorized::NullableColumn>(c);
```

Comment:
Why here must be NullableColumn?

Reply:
Because in the FE side, we can make sure this slot is nullable. Otherwise, this slot id won't appear in `_filter_null_value_columns`.

Reply:
I see your logic. But would better check in BE. for nullable slot, there would be not null column in BE.

Reply:
OK.